### PR TITLE
docs: Tweak FAQ

### PR DESCRIPTION
**Pull request by richb-hanover**
Update What can you do with PRQL to say... "Prototype PRQL queries...";
Update Press Materials link to point to PRQL/prql-brand;
Fix "What's with the 'aggregate'..." heading

### CODE REVIEW

**Comment by richb-hanover** [comment]
@aljazerzen What should happen to the `dbt-prql` reference? Thanks

File: web/website/content/faq.md [hunk 1]
```diff
@@ -6,11 +6,15 @@ Here are some of the most common questions we hear. Have something else you'd
...
 
-We're still early, and the opportunities for using PRQL are focused on two
-integrations:
+Use PRQL to build SQL queries today. In addition, we are focusing on several
+integrations. Here's how you can use PRQL:
 
+- **Prototype your PRQL queries** in the
+  [Playground](https://prql-lang.org/playground/) or the
+  [VS Code extension](https://marketplace.visualstudio.com/items?itemName=PRQL-lang.prql-vscode)
+  and copy/paste the resulting SQL into your database.
```
Comment (aljazerzen) [inline web/website/content/faq.md]:
```suggestion
  and copy/paste the resulting SQL into your database. It's not a great
  workflow, but it's easy to get started.
```

**Comment by aljazerzen** [comment]
> What should happen to the dbt-prql reference?

That can go out. It's not usable at the moment.

**Comment by richb-hanover** [comment]
>> What should happen to the dbt-prql reference?
>
> That can go out. It's not usable at the moment.

The current website lists `dbt-prql` and `Jupyter`. Are neither of these available now? Thanks.

**Comment by aljazerzen** [comment]
Jupyter works, dbt does not.

Can you also include this:

```
For more information about what's possible with PRQL, checkout
[Querying with PRQL](https://eitsupi.github.io/querying-with-prql/).
```

File: web/website/content/faq.md [hunk 2]
```diff
@@ -119,7 +121,9 @@ most dialects' implementation are identical to a generic implementation.
 
 {{</ faq >}}
 
-{{< faq "What's going on with this `aggregate` function? What's wrong with `SELECT` & `GROUP BY`?" >}}
+{{< faq "What's this `aggregate` function?" >}}
+
+**...and why not just use `SELECT` & `GROUP BY`?**
```
Comment (max-sixty) [inline web/website/content/faq.md:126]: Much better!

**Comment by max-sixty** [comment]
@richb-hanover sorry I lost track of these, I've gradually been getting my notifications back in order.

These are good improvements as ever, thank you!

**Pull request merged**